llvm/CMakeLists.txt

```diff
@@ -803,6 +803,12 @@ if(NOT LLVM_ENABLE_NEW_PASS_MANAGER)
                       " no longer supported.")
 endif()
 
+set(LLVM_ENABLE_OPAQUE_POINTERS FALSE CACHE BOOL
```

Comment:
What would you think about using a different name to make it clear that this variable is specific to the intel/llvm fork?

Reply:
I've renamed the option to `DPCPP_ENABLE_OPAQUE_POINTERS`.